pallets/utxo/src/script.rs

```diff
@@ -32,7 +32,7 @@ impl RawBlockTime {
     // Values less than this constant are considered to be a block number, other values are
     // interpreted as UNIX time stamp. This constant is identical to what is used in Bitcoin.
     // At 1min/block, the block numbers run out in about 900 years after the genesis block.
-    const THRESHOLD: u64 = 500_000_000u64;
+    pub const THRESHOLD: u64 = 500_000_000u64;
```

Comment:
I think you don't have to make this public if you make the tests under the `#[tests]`. This is a huge problem in C++, that you cannot access private variables in tests. But I think rust solved that problem. Please double check.

Reply:
I was thinking of something like:

```rust
#[cfg(test)]
const THRESHOLD: u64 = 500_000_000u64;
#[cfg(test)]
pub const THRESHOLD: u64 = 500_000_000u64;
```

But found it a bit too verbose for a small issue like this. If Rust has a better mechanism, that's great.

Reply:
So in the end, I moved the test helper function that uses the THRESHOLD constant into the `test` submodule of this module so it has access to struct fields without making them public. The function is then imported in `test.rs`.